front/scripts/i18n-api-errors.ts

```diff
@@ -65,19 +66,21 @@ async function checkI18N(error: any): Promise<string[]> {
 
 async function run() {
   try {
-    const api = await SwaggerParser.validate(openapi_path);
+    const api: OpenAPI.Document = await SwaggerParser.validate(openapi_path);
+
+    if (!('openapi' in api)) throw new Error('Expected an OpenAPI schema');
 
     // Do some checks on the generic error
-    if (!api.components.schemas.EditoastError)
-      throw new Error(`"EditoastError" can't be found in "components > schemas"`);
-    if (!api.components.schemas.EditoastError.oneOf)
+    const EditoastError = api?.components?.schemas?.EditoastError;
+    if (!EditoastError) throw new Error(`"EditoastError" can't be found in "components > schemas"`);
+    if (!('oneOf' in EditoastError) || !EditoastError.oneOf)
```

Comment:
This is painful 😮‍💨

Reply:
This is because `EditoastError` could be something like a `boolean`: OpenAPI allows schemas to be `{ "oneOf": … }` but also `true` or `{ "items": … }`.